rush.json

```diff
@@ -59,7 +59,7 @@
      * The default value is false to avoid legacy compatibility issues.
      * It is strongly recommended to set strictPeerDependencies=true.
      */
-    "strictPeerDependencies": true,
+    "strictPeerDependencies": false,

```

Comment:
For now we need to live with this a bit. Until we update the TS version.
https://github.com/puppeteer/puppeteer/issues/9698 
😢